package.json

```diff
@@ -198,16 +198,17 @@
     "onLanguage:html",
     "onLanguage:typescript"
   ],
-  "main": "./dist/client/src/extension",
+  "main": "./bazel-bin/client/src/extension",
```

Comment:
I think if you make this change you also have to update the substitution in the root BUILD file here:
https://github.com/angular/vscode-ng-language-service/blob/e808e621827f06fc68efe6fc1ba2f86fbfb70e3f/BUILD.bazel#L29

Reply:
Guessing "main" is load bearing on debugging somehow?

Reply:
👍 Done

Reply:
> Guessing "main" is load bearing on debugging somehow?

More or less, yes I think. It's more generic https://docs.npmjs.com/cli/v8/configuring-npm/package-json#main but probably has to do with how the extension development host runs the extension